3rdparty/find_dependencies.cmake

```diff
@@ -1193,16 +1193,18 @@ if(BUILD_GUI)
                                  clang-10
                                  clang-9
                                  clang-8
-                                 clang-7)
+                                 clang-7
```

Comment:
Finding libc++ location for all versions of llvm / clang is difficult to maintain. Perhaps we could assume that clang is in the path and run it to check it's version and query it for the location of libc++ and libc++abi? This would also let it work in Redhat based Linux, where the library folder is different.

Reply:
I tried `clang -print-search-dirs` on both Fedora and Ubuntu. They have slightly different behaviors. We can revisit this sometime later.